### DISCUSSION TITLE
Adding Presenter into MetaData

### DISCUSSION THREAD

**Discussion by dl-alexandre** [WWDCNotes/WWDCNotes#47] [Ideas]
I don't know how session data is obtained but can pages have presenter data either as part of the generated MetaData atop the page or as a standardized display similar to the @ Contributors info. I don't know whether stats similar to what contributors have would be desired or necessary, but having a standard of acknowledgment for presenter(s) would be a start.

**Comment by Jeehut** [comment]
@dl-alexandre Thank you for this suggestion. Personally, I'm open to highlight presenters more and maybe even auto-generate a page like we do right now for contributors. But I am not sure if this is something the presenters actually want.

If there was a way to ask the presenters and get their opinion on it, I'd be open to add this information to the metadata if they agree. But Apple themselves don't highlight the presenters at all in any way within the Developer app or WWDC session websites. The sessions have lots of metadata like description, keywords, even a transcript – but presenters are clearly missing even though that's an obvious thing they could have added. This is what I take as indication that Apple (or Apple employees) don't want presenters to be highlighted that much and the sessions should focus on the contents instead.

That's also why I always ask people to remove images with presenters when they add them to notes. I don't want to violate any personal rights. While I think Apple has an interest in this project overall as it helps the developer community (hence screenshots are generally OK), I have a feeling they might not like it if we start actively collecting info about the employees chosen as presenters.

Having that said, if you have an idea how we can get the presenters opinion on this topic, I'd be open to highlighting them more. Or if you have legal insights clearly saying that we can permanently keep their info online, please enlighten us. But there's no metadata we can automate regarding this topic and it would at least require some manual work.

**Comment by dl-alexandre** [comment]
I agree how you have it laid out, beyond the exposure Apple has presenters giving for some work they're doing. Separating their content from their person seems to be the right call. Although acknowledging their contribution can be just as important as the note itself whereas simply including their name does connect that reference in some way. I've inquired about this. I'll report what I learn. FB14212482
